.github/dependabot.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      # Keep the experimental modules up-to-date
+      - dependency-name: "github.com/grafana/xk6-*"
+        dependency-type: "all"
+    target-branch: "master"
+    commit-message:
+      prefix: "Bump "
```

Comment:
I think it already adds `Bump` as prefix by default 🤔 

Looking at an example of Dependabot commit, without `commit-message` configured, it looks like:
`Bump github.com/spf13/afero from 1.6.0 to 1.11.0`

Reply:
Would it make sense to make it a bit more descriptive instead? Something like `Update experimental module x/y/z from a.b.c to d.e.f`?